code_generator/cpp_testing/modules/codegen/kidl_test3/composition_type.yaml

```diff
@@ -18,6 +18,22 @@ enums:
     - enum_name: kpsr::codegen::OldEnum
       already_exists: true
       include_file: <klepsydra/codegen/old_enum.h>
+
+related_classes:
```

Comment:
As discussed, let's make it to work in a stub style, where we create stub yaml files for each related class.

Reply:
I've made this into a stub.